code/modules/mob/living/carbon/carbon.dm

```diff
@@ -516,7 +516,6 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 	return
 
 /mob/living/carbon/throw_item(atom/target)
-	throw_mode_off()
 	if(!target || !isturf(loc))
 		return
```

Comment:
The throw mode being missing from this return doesn't have any problematic effects, it's just a minor mechanical difference where instead of you losing your throw when the target/loc is bad you won't have to switch back to throw intent to give a second attempt